src/Features/LanguageServer/Microsoft.CodeAnalysis.LanguageServer/HostWorkspace/ProjectDependencyHelper.cs

```diff
@@ -0,0 +1,109 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Collections.Immutable;
+using System.Runtime.Serialization;
+using Microsoft.CodeAnalysis.LanguageServer.LanguageServer;
+using Microsoft.CodeAnalysis.MSBuild;
+using Microsoft.CodeAnalysis.PooledObjects;
+using Microsoft.Extensions.Logging;
+using NuGet.ProjectModel;
+using NuGet.Versioning;
+using Roslyn.Utilities;
+
+namespace Microsoft.CodeAnalysis.LanguageServer.HostWorkspace;
+internal static class ProjectDependencyHelper
+{
+    private const string UnresolvedProjectDependenciesName = "workspace/_roslyn_unresolvedProjectDependencies";
```

Comment:
I admit what this message was wasn't entirely obvious until digging into the code. even projectHasUnresolvedDependencies would make it a bit clearer this was to let the client know that. Or a comment would be good.